content/en/platform/corda/5.0-beta/managing-application-networks/plan.md

```diff
@@ -0,0 +1,12 @@
+---
+date: '2023-02-23'
+title: "Planning Application Networks"
+menu:
+  corda-5-beta:
+    identifier: corda-5-beta-app-networks-plan
+    parent: corda-5-beta-app-networks
+    weight: 1000
+section_menu: corda-5-beta
+---
+
+_How to plan and design your application Network_
```

Comment:
Is this supposed to be empty .md ?

Reply:
We're still waiting on the draft content for this section but I wanted to include it because part of the review is to confirm that people are happy with the full hierarchy and flow of the new section.